### PR TITLE
[zh] Localize ist0166 into Chinese

### PR DESCRIPTION
**Pull request by windsonsea**
Sync with en #14365 

- [ ] Ambient
- [x] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Extensions and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure
- [x] Localization/Translation

### CODE REVIEW

File: content/zh/docs/reference/config/analysis/ist0166/index.md [hunk 1]
```diff
@@ -0,0 +1,103 @@
+---
+title: IneffectiveSelector
+layout: analysis-message
+owner: istio/wg-user-experience-maintainers
+test: n/a
+---
+
+当 `AuthorizationPolicy`、`RequestAuthentication`、`Telemetry` 或 `WasmPlugin`
+这类策略中的工作负载选择器在 Kubernetes Gateway 中没有有效指向任何 Pod 目标时，会出现此消息。
```
Comment (hanxiaop) [inline content/zh/docs/reference/config/analysis/ist0166/index.md:9]:
```suggestion
这类策略中的工作负载选择器没有有效指向任何 Kubernetes Gateway Pod 目标时，会出现此消息。
```

Comment (windsonsea) [reply content/zh/docs/reference/config/analysis/ist0166/index.md]: Thanks for your careful review. Updated.

File: content/zh/docs/reference/config/analysis/ist0166/index.md [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+title: IneffectiveSelector
+layout: analysis-message
+owner: istio/wg-user-experience-maintainers
+test: n/a
+---
+
+当 `AuthorizationPolicy`、`RequestAuthentication`、`Telemetry` 或 `WasmPlugin`
+这类策略中的工作负载选择器没有有效指向任何 Kubernetes Gateway Pod 目标时，会出现此消息。
+
+## 示例 {#Example}
```
Comment (wilsonwu) [inline content/zh/docs/reference/config/analysis/ist0166/index.md:11]:
```suggestion
## 示例 {#example}
```

**Review by wilsonwu — APPROVED** [review]
LGTM, thanks!

**Pull request merged**